internal/kafka/command_share_group.go

```diff
@@ -0,0 +1,52 @@
+package kafka
+
+import (
+	"github.com/spf13/cobra"
+
+	pcmd "github.com/confluentinc/cli/v4/pkg/cmd"
+)
+
+type shareGroupOut struct {
+	Cluster            string `human:"Cluster" serialized:"cluster"`
+	ShareGroup         string `human:"Share Group" serialized:"share_group"`
+	Coordinator        string `human:"Coordinator" serialized:"coordinator"`
+	State              string `human:"State" serialized:"state"`
+	ConsumerCount      int32  `human:"Consumer Count" serialized:"consumer_count"`
+	PartitionCount     int32  `human:"Partition Count" serialized:"partition_count"`
+	TopicSubscriptions string `human:"Topic Subscriptions" serialized:"topic_subscriptions"`
+}
+
+// shareGroupListOut is used specifically for the list command to exclude partition count
+type shareGroupListOut struct {
+	Cluster       string `human:"Cluster" serialized:"cluster"`
+	ShareGroup    string `human:"Share Group" serialized:"share_group"`
+	Coordinator   string `human:"Coordinator" serialized:"coordinator"`
+	State         string `human:"State" serialized:"state"`
+	ConsumerCount int32  `human:"Consumer Count" serialized:"consumer_count"`
+}
```

Comment:
Since this is a subset of the other `shareGroupOut` struct, we can just use that one.

And on the list commands, you can filter to include only the used fields:
```
list.Filter([]string{"Cluster", "ShareGroup", "Coordinator", "State", "ConsumerCount"})
```